### PR TITLE
Gradle Plugin Change Logs

### PR DESCRIPTION
**Pull request by Ithildir**

### CODE REVIEW

**Comment by Ithildir** [comment]
ci:stop

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/2d383dd8a6...2faa34fba0

File: modules/sdk/gradle-plugins-source-formatter/CHANGELOG.markdown [hunk 1]
```diff
@@ -1083,6 +1083,18 @@ another task with the same name is being executed in a parent project.
...
+
+### Added
+- [LPS-74314]: Add the property `fileExtensions` to `FormatSourceTask`.
+
+### Changed
+- [LPS-74314]: Update the [Liferay Source Formatter] dependency to version
+1.0.471.
+
+### Fixed
+- [LPS-74314]: Fix passing of the `baseDir` property to Source Formatter.
```
Comment (codyhoag) [inline modules/sdk/gradle-plugins-source-formatter/CHANGELOG.markdown]: @Ithildir What are we passing the `baseDir` property from? The Gradle plugin?

Comment (Ithildir) [reply modules/sdk/gradle-plugins-source-formatter/CHANGELOG.markdown]: The `baseDir` is a property of `FormatSourceTask`, which means it's a property of the `formatSource` Gradle task and all other Gradle tasks of type `FormatSourceTask`. Then, when the task runs, the value of the `baseDir` property is passed to SF itself via command line arguments.

The Gradle plugin sets a default value of the `baseDir` property, but people can change it.

Comment (codyhoag) [reply modules/sdk/gradle-plugins-source-formatter/CHANGELOG.markdown]: @Ithildir does this sound OK/accurate?

> Fix the process of passing the `baseDir` property from the `FormatSourceTask` to the Source Formatter.

Comment (Ithildir) [reply modules/sdk/gradle-plugins-source-formatter/CHANGELOG.markdown]: Yes, it sounds very good! Thanks!!

**Pull request closed without merging**